ibis/expr/timecontext.py

```diff
@@ -252,7 +252,7 @@ def construct_time_context_aware_series(
 
 @functools.singledispatch
 def adjust_context(
-    op: Any, timecontext: TimeContext, scope: Optional['Scope'] = None
+    op: Any, timecontext: TimeContext, *, scope: Optional['Scope']
```

Comment:
Slightly weird that `scope` is kw-only but `timecontext` is not. Not a big deal though.